data-tool/flows/tombstone/tombstone_queries.py

```diff
@@ -757,6 +757,22 @@ def get_offices_held_query(corp_num):
     return query
 
 
+def get_cont_out_query(corp_num):
+    query = f"""
+    select 
+        co.can_jur_typ_cd,
+        to_char(co.cont_out_dt, 'YYYY-MM-DD HH24:MI:SSTZH:TZM') as cont_out_dt,
+        co.othr_juri_desc,
+        co.home_company_nme
+    from cont_out co
+        join corp_state cs on cs.corp_num = co.corp_num and cs.end_event_id is null
+    where co.corp_num = '{corp_num}'
+      and co.end_event_id is null
+      and cs.state_type_cd in ('HCO', 'HAO')
```

Comment:
Do we map amalgamate out to continuaion out? Is it decided?

Reply:
Not decided. Most probably will reuse continuation out